tests/test_5_mut.py

```diff
@@ -426,3 +427,24 @@ def test_non_C_nparray():
     non_standard_stride = np.asfortranarray(points)
     section.points = non_standard_stride
     assert_array_equal(section.points, points)
+
+def test_annotation():
+    with captured_output() as (_, err):
+        with ostream_redirect(stdout=True, stderr=True):
+            with tmp_asc_file('''((Dendrite)
+                      (3 -4 0 2)
+                      (3 -6 0 2)
+                      (3 -8 0 2)
+                      (3 -10 0 2)
+                      (
+                        (3 -10 0 2)
+                        (0 -10 0 2)
+                        (-3 -10 0 2)
+                        |       ; <-- empty sibling but still works !
```

Comment:
is the annotation?

Reply:
No, no. The annotation is just that there is a single child. This is just an additionnal comment.